moropy_cli/.gitignore

```diff
@@ -0,0 +1,248 @@
+
```

Comment:
I don't understand the need for adding this. Don't we already have a gitignore in root directory?

Reply:
My git was not ignoring the `.idea` folder, so I thought I should create another in the subdirectory and this worked. Is there any way I can let my git know that it has to ignore the `.idea` folder?